libs/acn/E133StatusHelper.cpp

```diff
@@ -142,5 +142,104 @@ string ConnectStatusCodeToString(E133ConnectStatusCode connect_status_code) {
   }
   return "Unknown E1.33 Connect Status Code";
 }
+
+
+bool IntToRPTStatusCode(uint16_t input,
+                        RPTStatusVector *rpt_status_code) {
+  switch (input) {
```

Comment:
rpt_status_code could also use a null check here

Reply:
Tests and null check added.